projects/wave-core-new/src/lib/datasets/upload/upload.component.scss

```diff
@@ -18,5 +18,5 @@ mat-spinner {
 }
 
 textarea {
-    width: 100%;
+    width: 99%;
```

Comment:
What is this about?

Reply:
If I leave it at 100% I get a scrollbar, because the textarea is now inside a tabcomponent. Not sure how to properly fix this.